client/src/App.css

```diff
@@ -266,5 +266,5 @@ body {
   background-color: rgb(175, 175, 95);
 }
 .rota_container2 {
-  background-color: rgb(243, 243, 172);
+  background-color: var(--yellow-green);
```

Comment:
Hey @lalehkoupa this looks good! However, I am just wondering if we start using variables, should we start using SASS?
https://sass-lang.com/documentation/variables